recipes/r-chebpol/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '2.1-2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-chebpol
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/chebpol_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/chebpol/chebpol_{{ version }}.tar.gz
+  sha256: f17f23a4e038f7cc7a5853710e5954019316426ee9710fb018c3cb442d7ecc88
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  ignore_run_exports: r-geometry
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-geometry
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-geometry
+    - libopenblas
```

can this be libblas so that blas is switchable, see https://conda-forge.org/docs/maintainer/knowledge_base.html#blas
```suggestion
    - libblas
```

---

@dbast looks like this fails for linux and osx. Can you provide some guidance here? I assume I need to unpin libblas from a specific vendor or otherwise add a platform filter. Thanks
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=293714&view=results